test/builtin_dataset_mocks.py

```diff
@@ -799,8 +822,11 @@ def add_bndbox(obj):
     def generate(cls, root, *, year, trainval):
         archive_folder = root
         if year == "2011":
-            archive_folder /= "TrainVal"
-        data_folder = archive_folder / "VOCdevkit" / f"VOC{year}"
+            archive_folder = root / "TrainVal"
+            data_folder = archive_folder / "VOCdevkit"
+        else:
+            archive_folder = data_folder = root / "VOCdevkit"
+        data_folder = data_folder / f"VOC{year}"
```

Comment:
Is this related?

Reply:
Yes this is related if we want to enforce that no extra files are generated. See https://github.com/pytorch/vision/pull/5990#pullrequestreview-970563847 for details.